docs/data-ai/data/visualize.md

```diff
@@ -4,6 +4,269 @@ title: "Visualize data"
 weight: 20
 layout: "docs"
 type: "docs"
+images: ["/services/icons/data-visualization.svg"]
+icon: true
+aliases:
+  - /data/visualize/
+  - /use-cases/sensor-data-visualize/
+  - /how-tos/sensor-data-visualize/
 no_list: true
-description: "TODO"
+viamresources: ["sensor", "data_manager"]
+platformarea: ["data", "fleet"]
+date: "2024-12-04"
+description: "Use teleop or grafana to visualize sensor data from the Viam app."
 ---
+
+Once you have used the data management service to [capture data](/data-ai/data/get-started/capture-sync/), you can visualize your data with either the Viam app's **TELEOP** page or a variety of third-party tools, including Grafana, Tableau, Google's Looker Studio, and more.
+
+<!-- You can choose to visualize data from a component on one machine, from multiple components together, or from many components across a fleet of machines.
+
+For example, you can configure data capture for several sensors across multiple machines to report the ambient operating temperature.
+You can then visualize that data to easily understand how the ambient temperature affects your machines' operation.
+
+You can do all of this using the [Viam app](https://app.viam.com/) user interface. You will not need to write any code. -->
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Visualize data with teleop](#prerequisites-teleop)
+1. [Visualize data with 3rd party tools](#prerequisites-3rd-party-tools)
+
+{{< /alert >}}
+
+## Prerequisites: Teleop
+
+{{% expand "A configured machine with sensor components" %}}
+
+Make sure your machine has at least one of the following:
+
+- A movement sensor or sensor
+
+See [configure a machine](/how-tos/configure/) for more information.
+
+{{% /expand%}}
+
+## Configure a sensor widget
+
+Follow our docs to [configure a teleop workspace](manage/troubleshoot/teleoperate/).
```

Comment:
You'll want to have the instructions here instead, copy them or create an include

Reply:
copied over bc needed some adjustments